install_config/topics/ocp_osp_enabling_octavia.adoc

```diff
@@ -111,3 +111,10 @@ backend is ML2/OVS. There is no need for modifications if the backend is
 ML2/OVN.
 ====
 
+[NOTE]
+====
+If Kuryr SDN is used, ML2/OVS is the preferred Neutron backend at the moment.
+ML2/OVN is also supported but due to an issue that impacts the performance
```

Comment:
I'd reword this sentence. If the creation doesn't timeout, then what about:

> ML2/OVN is also supported. Container creation may take longer than expected due to performance issues.

If container creation eventually does timeout, then I would say:

While ML2/OVN is also supported, there are known issues that can result in a timeout during container creation.

Reply:
Agree